_infra/helm/locust/locustfiles/locustfile.py

```diff
@@ -478,19 +478,31 @@ def post(self, url: str, data: dict = {}):
 class FrontstageTasks(TaskSet, Mixins):
 
     def on_start(self):
-        self.sign_in()
+        requests_filepath = "requests.json"
```

Comment:
Feels like a constant, and something that we might want to override

Reply:
I have created an environment variable and assigned the filepath to that variable so it can be called.

Reply:
Moved to a global